client_app/src/components/shelter/ShiftDetails.js

```diff
@@ -69,24 +82,29 @@ export const ShiftDetails = () => {
               <DatePicker defaultValue={dayjs()} slotProps={{textField: {size: "small"}}} sx={{width: 200}} />
             </LocalizationProvider>
             <h4 className="endtime-label">Select Shift(s): </h4>
-            <Box sx={{ minWidth: 120 }}>
-              <FormControl fullWidth>
-                <InputLabel id="demo-simple-select-label">Shift(s)</InputLabel>
-                <Select
-                  labelId="demo-simple-select-label"
-                  id="demo-simple-select"
-                  value={selectedShifts}
-                  label="Shift(s)"
-                  onChange={handleChange}
-                >
-                  {renderDropDown()}
-                </Select>
-              </FormControl>
-            </Box>
+            <FormControl sx={{ m: .5, width: 250}}>
+              <InputLabel id="shiftsCheckedDropDown" hidden={isEmergencyModalOpen||isStatusModalOpen||isVolunteerModalOpen}>Shift(s)</InputLabel>
+              <Select
+                labelId="shiftCheckboxLabel"
+                id="multiShiftCheckboxLabel"
+                multiple
+                value={selectedShifts}
+                onChange={handleChange}
+                input={<OutlinedInput label="Shift(s)" />}
+                renderValue={(selected) => selected.join(', ')}
+              >
+                {renderDropDown()}
+              </Select>
+            </FormControl>
           </div> 
         </div>
         <div className="shiftdetails-table">
-          <ShiftDetailsTable onSignUpVolunteersClick={onSignUpVolunteersClick} onSendEmergencyAlertClick={onSendEmergencyAlertClick} onStatusModalClick={onStatusModalClick}/>
+          <ShiftDetailsTable 
+            onSignUpVolunteersClick={onSignUpVolunteersClick} 
+            onSendEmergencyAlertClick={onSendEmergencyAlertClick} 
+            onStatusModalClick={onStatusModalClick}
+            stickyHeader={stickyHeader}
+            headerZIndex={headerZIndex}/>
         </div>
       </div>
     );
```

Comment:
I think adding the stick columns, as discussed, wouldnt be useful because it would take up the entire screen. I like the use of the sticky header here. I dont think you need to make any UI adjustments in regard to the headers/columns